frontend/src/components/ResourceCard/styles.scss

```diff
@@ -1,22 +1,130 @@
-.collapsed {
-    border: solid rgb(201, 194, 194) 2.5px;
-    border-radius: 7px;
+.collapsed, .expanded {

```

Comment:
What do you think about us encapsulating everything in this file in the class name, i.e. `.resource-card`?

With SCSS, nesting becomes super simple, since the following written in SCSS:
```css
.a {
  .b {
     property: left;
  }
  .c {
     property: right;
  }
}
```
Becomes, in CSS: 
```css
.a .b {
     property: left;
}
.a .c {
     property: right;
}
```

So all we have to do is wrap everything in, say, 
```css
.resource-card { 
   /* css here... */
}
```
and wrap the resource card component in a `div`.

This can help us keep the global CSS namespace clean, so that if we want to use, say, the class `.collapsed`, it won't automatically take on the special property defined by the below selector.

Reply:
I agree, I will make these changes